### DISCUSSION TITLE
ΓΕ 3 - Μέρος Α - Συνεννόηση για την επιλογή μελετών περίπτωσης

### DISCUSSION THREAD

**Discussion by afoscolou** [sdy60-2022/.github#41] [Ideas]
Γνωρίζω ότι είναι λίγο πρόωρο και οι περισσότεροι μόλις τελειώσατε με την παράδοση της ΓΕ 2 αλλά, επειδή κάποιοι ίσως αρχίσουν να ασχολούνται με την 3η εργασία μέσα στις γιορτές, ανοίγω αυτό το thread για να γίνει και πάλι κάποιος συντονισμός μεταξύ μας ώστε να μην ανεβάσουμε τις ίδιες μελέτες περίπτωσης. Η προηγούμενη προ-συνεννόηση δούλεψε μια χαρά, οπότε όποιος βρίσκει τι θέλει να χρησιμοποιήσει, μπορεί να το ποστάρει σε αυτή την συζήτηση προς ενημέρωση των υπολοίπων.

Καλές γιορτές σε όλους και όλες και καλή ξεκούραση!

**Comment by sotkot** [comment]
Καλησπέρα σε όλους!

Θα ασχοληθώ με το Microsoft HoloLens σαν Μελέτη Περίπτωσης.

**Comment by afoscolou** [comment]
Γεια σας.

Σχετικά με την Μελέτη Περίπτωσης επιλέγω το Google Glass.

**Reply by vkaragia** [reply]
Καλησπέρα σας , εγώ θα ασχοληθώ με στρατιωτικές εφαρμογές , συγκεκριμένα με το JHMCS

**Comment by atsiano81** [comment]
Καλησπέρα θα ανεβάσω για το HTC VIVE
